tests/flytekit/unit/core/test_conditions.py

```diff
@@ -495,3 +496,41 @@ def multiplier_2(my_input: float) -> float:
 
     res = multiplier_2(my_input=10.0)
     assert res == 20
+
+
+def test_echo_in_condition():
+    echo1 = Echo(name="echo", inputs={"a": float})
+
+    @task()
+    def t1(radius: float) -> typing.Optional[float]:
+        return 2 * 3.14 * radius
+
+    @workflow
+    def wf1(radius: float) -> typing.Optional[float]:
+        return (
+            conditional("shape_properties_with_multiple_branches")
+            .if_((radius >= 0.1) & (radius < 1.0))
+            .then(t1(radius=radius))
+            .else_()
+            .then(echo1(a=radius))
+        )
+
+    assert wf1(radius=1.8) == 1.8
+
+    echo2 = Echo(name="echo", inputs={"a": float, "b": float})
+
+    @task()
+    def t2(radius: float) -> (float, float):
+        return 2 * 3.14 * radius, 2 * 3.14 * radius
+
+    @workflow
+    def wf2(radius1: float, radius2: float) -> (float, float):
```

Comment:
I do not think a typing with a tuple like this is support in Python: https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#useful-built-in-types

```suggestion
    def wf2(radius1: float, radius2: float) -> typing.Tuple[float, float]:
```

Reply:
Updated it, thanks. I saw we use `-> (...)` in many places in flytekit.